src/views/AnalysisModule/Analysis/index.tsx

```diff
@@ -16,16 +32,216 @@ interface ComponentProps {
     title: string;
     startDate?: string;
     endDate?: string;
+    activeProject: number;
+    analysisId: number;
+    setAnalysisDeleteId: (value: number) => void;
+    analyses: AnalysisElement;
+    teamLeadName: string;
+}
+
+interface AnalysisPillars {
+    id: number;
+    assigneeName: string;
+    title: string;
+    mainStatement: string;
+    informationGap: string;
+    filters?: string;
+    analysis: number;
+}
+
+interface AnalysisPillarRendererProps extends Omit<AnalysisPillars, 'id'> {
+    pillarId: AnalysisPillars['id'];
+    setDeleteId: (value: number) => void;
+}
+
+interface PillerListRendererProps extends Omit<AnalysisPillars, 'id'>{
+    pillarId: AnalysisPillars['id'];
 }
 
+const mapStateToProps = (state: AppState) => ({
+    activeProject: activeProjectIdFromStateSelector(state),
+});
+
+const MAX_ITEMS_PER_PAGE = 50;
+function AnalysisPillarRenderer(props: AnalysisPillarRendererProps) {
+    const {
+        pillarId,
+        title,
+        assigneeName,
+        setDeleteId,
+    } = props;
+
+    const [isCompleted, setIsCompleted] = useState(false);
+    // setIsCompleted to be used when the status is passed by API
+
+    const handleDeletePillar = useCallback(() => {
+        setDeleteId(pillarId);
+    }, [pillarId, setDeleteId]);
+
+    return (
+        <Container
+            className={styles.pillar}
+            heading={(
+                <div className={styles.left}>
+                    {title}
+                    <Tag
+                        className={styles.tag}
+                        variant={isCompleted ? 'accent' : 'gradient1'}
+                    >
+                        {isCompleted
+                            ? _ts('analysis', 'completeLabel')
+                            : _ts('analysis', 'inProgressLabel')
+                        }
+                    </Tag>
+                </div>
+            )}
+            headerClassName={styles.heading}
+            headerActions={(
+                <div className={styles.headerRight}>
+                    <QuickActionButton
+                        className={styles.button}
+                    >
+                        <Icon name="edit" />
+                    </QuickActionButton>
+                    <QuickActionButton
+                        className={styles.button}
+                    >
+                        <Icon name="copy" />
+                    </QuickActionButton>
+                    <QuickActionButton
+                        className={styles.button}
+                        onClick={handleDeletePillar}
+                    >
+                        <Icon name="delete" />
+                    </QuickActionButton>
+                </div>
+            )}
+            headerDescription={(
+                <div className={styles.subHeading}>
+                    <span className={styles.boldText}>
+                        {_ts('analysis', 'creationDate')}
+                    </span>
+                    June 5, 2021
```

Comment:
Fetch this from the API as well and use Fromatted Date component

Reply:
There is no date in analysis pillar. Or should we use the date of creation of the analysis instead?